script.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'b6ad49d4';
```

Comment:
In case too many result the 2-nd div should not be there

<img width="927" alt="Screenshot 2022-11-18 at 11 41 19" src="https://user-images.githubusercontent.com/34886027/202671036-6efcf4fb-0156-4eac-8d18-bf676f107836.png">